include/simdjson/dom/element.h

```diff
@@ -336,8 +336,8 @@ class element {
    * The key will be matched against **unescaped** JSON:
    *
    *   dom::parser parser;
-   *   parser.parse(R"({ "a\n": 1 })")["a\n"].get<uint64_t>().value == 1
-   *   parser.parse(R"({ "a\n": 1 })")["a\\n"].get<uint64_t>().error == NO_SUCH_FIELD
+   *   parser.parse(R"({ "a\n": 1 })")["a\n"].get<uint64_t>().first == 1
```

Comment:
The switch to first/second is somewhat unfortunate (as opposed to value(), error()) because it forces the user to remember who is on first and who is on second. (joke)

Reply:
But thinking about it, you'd probably not want to ever call it like so.

Reply:
I'm not happy about .first, but it's not really something we want people to use outside of tests like this ... it's a method to ignore errors when exceptions are off.

Reply:
What is wrong with...
```
parser.parse(R"({ "a\n": 1 })"_padded)["a\n"].get<uint64_t>().value();
```
???

Reply:
> What is wrong with...
> 
> ```
> parser.parse(R"({ "a\n": 1 })"_padded)["a\n"].get<uint64_t>().value();
> ```
> 
> ???

.value() presently only exists when exceptions are off, because it throws an exception if there was an error. We could totally make it work for non-exception cases by removing the exception throw and treating it like it means `.assume_no_error()`. When I built value() a while back, I was being cautious about introducing methods that help you ignore errors.

Reply:
> When I built value() a while back, I was being cautious about introducing methods that help you ignore errors. 

That sounds reasonable.